e2e/rbd.go

```diff
@@ -189,12 +189,26 @@ func validateRBDImageCount(f *framework.Framework, count int, pool string) {
 		framework.Failf("failed to list rbd images: %v", err)
 	}
 	if len(imageList) != count {
+		var imageDetails []string // To collect details for all images
+		for _, image := range imageList {
+			imgInfo, err := getImageInfo(f, image, pool)
+			if err != nil {
+				return
+			}
+			imgStatus, err := getImageStatus(f, image, pool)
+			if err != nil {
+				return
+			}
+			// Collecting image details for printing
+			imageDetails = append(imageDetails, fmt.Sprintf("Pool Name: %s, Image Name: %s, Img Info: %v, Img Status: %v", pool, image, imgInfo, imgStatus))
```

Comment:
golangci-lint complains that this line is too log: https://github.com/ceph/ceph-csi/actions/runs/11613610012/job/32379913964?pr=4938#step:3:820

Reply:
done